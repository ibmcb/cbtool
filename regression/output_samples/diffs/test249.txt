 status: vm_24 (cloud-assigned uuid F1FFDF76-94AA-5941-8A9B-1575C3D3840D) could not be created on SimCloud "TESTCLOUD" : Failed to create VM image (The VM creation was rolled back)
 status: Sending a termination request for vm_24 (cloud-assigned uuid F1FFDF76-94AA-5941-8A9B-1575C3D3840D)....
VM object 40B34E12-5D7B-55AE-B160-47FE844182CA (named "vm_24") could not be attached to this experiment: vm_24 (cloud-assigned uuid F1FFDF76-94AA-5941-8A9B-1575C3D3840D) could not be created on SimCloud "TESTCLOUD" : Failed to create VM image (The VM creation was rolled back)
|Used Memory                                          |19.80M                        
|Uptime (in seconds)                                  |2516531                       
|Total Connections Received                           |185643                        
|Total Commands Processed                             |11600000                      
|Storage Size                                         |18350080                      
|Data Size                                            |6620572                       
|Index Size                                           |2101232                       
|Average Object Size                                  |613.868521094                 
|AD93E38D-EAEC-5C15-BA20-7A27F63D13FF (simzone_b) VM RESERVATIONS|0                             
|E4B581D8-2DD3-5956-8196-B1B60C5DC5B0 (simzone_d) VM RESERVATIONS|0                             
|AD29082C-1389-58A3-BE64-3784D075F191 (simzone_c) VM RESERVATIONS|0                             
|75A875C6-F2C5-5C3D-BD34-051FBBE24C3A (simzone_a) VM RESERVATIONS|0                             
