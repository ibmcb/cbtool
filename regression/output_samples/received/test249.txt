 status: vm_24 (cloud-assigned uuid F1FFDF76-94AA-5941-8A9B-1575C3D3840D) could not be created on SimCloud "TESTCLOUD" : Failed to create VM image (The VM creation was rolled back)
 status: Sending a termination request for vm_24 (cloud-assigned uuid F1FFDF76-94AA-5941-8A9B-1575C3D3840D)....
VM object 40B34E12-5D7B-55AE-B160-47FE844182CA (named "vm_24") could not be attached to this experiment: vm_24 (cloud-assigned uuid F1FFDF76-94AA-5941-8A9B-1575C3D3840D) could not be created on SimCloud "TESTCLOUD" : Failed to create VM image (The VM creation was rolled back)
The following VMs are attached to this experiment (Cloud TESTCLOUD) :
No objects available.
The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |19.80M                        
|Redis Version                                        |2.6.2                         
|Uptime (in seconds)                                  |2516531                       
|Total Connections Received                           |185643                        
|Total Commands Processed                             |11600000                      
|Number of Keys                                       |324                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |18350080                      
|Data Size                                            |6620572                       
|Index Size                                           |2101232                       
|Average Object Size                                  |613.868521094                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ARRIVED                                          |22                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |22                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |2                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |134                           

 ---------------- VMC-WIDE COUNTERS ----------------
|AD93E38D-EAEC-5C15-BA20-7A27F63D13FF (simzone_b) VM RESERVATIONS|0                             
|E4B581D8-2DD3-5956-8196-B1B60C5DC5B0 (simzone_d) VM RESERVATIONS|0                             
|AD29082C-1389-58A3-BE64-3784D075F191 (simzone_c) VM RESERVATIONS|0                             
|75A875C6-F2C5-5C3D-BD34-051FBBE24C3A (simzone_a) VM RESERVATIONS|0                             

