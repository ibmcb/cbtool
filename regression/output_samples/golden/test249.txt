 status: vm_24 (cloud-assigned uuid 051EEA9F-7975-55D9-8315-D17E83C40788) could not be created on SimCloud "TESTCLOUD" : Failed to create VM image (The VM creation was rolled back)
 status: Sending a termination request for vm_24 (cloud-assigned uuid 051EEA9F-7975-55D9-8315-D17E83C40788)....
VM object F76A89B8-6295-51B6-9B72-D531FC04D4AD (named "vm_24") could not be attached to this experiment: vm_24 (cloud-assigned uuid 051EEA9F-7975-55D9-8315-D17E83C40788) could not be created on SimCloud "TESTCLOUD" : Failed to create VM image (The VM creation was rolled back)
The following VMs are attached to this experiment (Cloud TESTCLOUD) :
No objects available.
The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |19.87M                        
|Redis Version                                        |2.6.2                         
|Uptime (in seconds)                                  |2511818                       
|Total Connections Received                           |185365                        
|Total Commands Processed                             |11080144                      
|Number of Keys                                       |324                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |18321408                      
|Data Size                                            |5879612                       
|Index Size                                           |1937712                       
|Average Object Size                                  |619.558693361                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ARRIVED                                          |22                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |22                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |2                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |134                           

 ---------------- VMC-WIDE COUNTERS ----------------
|C4E90296-1043-55C8-8270-B5D7C3C38270 (simzone_b) VM RESERVATIONS|0                             
|2AEE56D4-E6A4-5FB2-8D29-4268D4C22AD0 (simzone_a) VM RESERVATIONS|0                             
|5F641DA9-DE4B-538C-B0CD-42A4F7D43E9E (simzone_d) VM RESERVATIONS|0                             
|A796D59F-B618-5C91-8FC6-F6F093DE22C5 (simzone_c) VM RESERVATIONS|0                             

